cmd/main.go

```diff
@@ -36,7 +36,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 
 	appprojectv1alpha1 "github.com/argoproj-labs/ephemeral-access/api/argoproj/v1alpha1"
-	ephemeralaccessv1alpha1 "github.com/argoproj-labs/ephemeral-access/api/v1alpha1"
+	ephemeralaccessv1alpha1 "github.com/argoproj-labs/ephemeral-access/api/ephemeral-access/v1alpha1"
```

Comment:
Is it necessary to move to a new folder? it is kind of intuitive to have the current project type in the api folder

Reply:
Seems like the required structure by kubebuilder 😞

Reply:
It is required for multi-group project layouts. I kind of like the proposed layout :)